conda-recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ build:
 
   script:
     - cd $RECIPE_DIR/..
-    - $PYTHON setup.py install
+    - LDFLAGS="-shared" FFLAGS=-fPIC $PYTHON setup.py install # [not win]
```

Comment:
Once this works, I will try to get slycot + control in conda-forge

There might be something in the build tools that is not set right, but I don't know where to look for that.